rtdata/languages/default

```diff
@@ -1488,6 +1488,7 @@ HISTORY_MSG_ICM_WORKING_PRIM_METHOD;Primaries method
 HISTORY_MSG_ICM_WORKING_SLOPE;TRC - Slope
 HISTORY_MSG_ICM_WORKING_TRC_METHOD;TRC method
 HISTORY_MSG_ILLUM;CAL - SC - Illuminant
+HISTORY_MSG_LOCAL_AVOIDNEGATIVE;Local - SC - Avoid negatives values
```

Comment:
```suggestion
HISTORY_MSG_LOCAL_AVOIDNEGATIVE;Local - SC - Avoid negative values
```

Reply:
I will push "Avoid zero and negative values"